assets/queries/crossplane/gcp/cloud_storage_bucket_logging_not_enabled/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "6c2d627c-de0f-45fb-b33d-dad9bffbb421",
+  "queryName": "Cloud Storage Bucket Logging Not Enabled",
+  "severity": "HIGH",
+  "category": "Observability",
+  "descriptionText": "Cloud storage bucket with logging not enabled",
```

Improve the description, please. The same applies to the related queries

```suggestion
"descriptionText": "Cloud storage bucket should have logging enabled",
```